ssl/ssl_test.cc

```diff
@@ -399,6 +399,105 @@ static const CipherTest kCipherTests[] = {
     },
 };
 
+// In kTLSv13RuleOnly, |rule| of each CipherTest can only match TLSv1.3 ciphers.
+// e.g. kTLSv13RuleOnly does not have rule "AESGCM+AES128", which can match
+// some TLSv1.2 ciphers.
+static const CipherTest kTLSv13RuleOnly[] = {
+    // Selecting individual ciphers should work.
+    {
+        "TLS_AES_256_GCM_SHA384:"
+        "TLS_CHACHA20_POLY1305_SHA256:"
+        "TLS_AES_128_GCM_SHA256",
+        {
+            {TLS1_CK_AES_256_GCM_SHA384, 0},
+            {TLS1_CK_CHACHA20_POLY1305_SHA256, 0},
+            {TLS1_CK_AES_128_GCM_SHA256, 0},
+        },
+        false,
+    },
+    // + reorders selected ciphers to the end, keeping their relative order.
```

Comment:
This extension is expected and does not hurt anything because the existing rule str process logics are reused. I don't plan to write another `rule_str` process logics given only 3 TLSv1.3 ciphers are supported.